src/types/list.rs

```diff
@@ -178,6 +178,42 @@ where
     }
 }
 
+macro_rules! array_impls {
+    ($($N:expr),+) => {
+        $(
+            impl<T> ToPyObject for [T; $N]
```

Comment:
As these imples are not faster than one for slices(`impl ~ for &[T]`), I don't think we need this.

Reply:
I agree for `ToPyObject` but not for `IntoPy<PyObject>` - see below.